### ISSUE TITLE
Some questions about the style control 

### ISSUE THREAD

**Issue by Bin-Ji** [simonalexanderson/StyleGestures#45]
Hi, thank you for providing the code, it really helps me a lot.
As proposed in the paper, four lower-level style attributes are discussed here. I wonder: 1. Why don't you use a four-dimensional vector 's_t' to control all four style attributes simultaneously; 2. Can I control a single style property with a multiple dimensional style input vector 's_t'?

**Comment by ghenter** [comment]
I'm glad to hear that our code has helped you.

> 1. Why don't you use a four-dimensional vector 's_t' to control all four style attributes simultaneously

Doing this is entirely possible, and covered by the general method described in our paper. In fact, it is one of the reasons that we wrote **s**_t as a vector in our text and figures. The different things and control setups we considered in our experiments (such as the four specific style attributes) are mere examples of the general formulation we propose.

Our experiments in the StyleGestures paper did not study simultaneous control of the four different style attributes from the experiments for several reasons:
1. There were constraints on the time available to us.
2. We were already running out of space for content in the article, and were having to delete valuable parts of our article because it was too long to be accepted.
3. Simultaneous control over multiple style aspects seemed more difficult to evaluate well and in a principled manner, since the value(s) of one attribute might affect the efficacy of control for another attribute. By limiting the evaluation to one-dimensional control, we could make clearer plots and visualisations like those in Fig. 4 of the paper.
4. We think the article still is a strong article even without also including control over multiple style aspects in the experiments.

> 2. Can I control a single style property with a multiple dimensional style input vector 's_t'?

Yes, you absolutely can. Such parametrisations are common practice in deep learning, for instance to represent different individual voices in a "speaker space" in text-to-speech. In fact, what you describe is another reason that we wrote **s**_t to in our paper to emphasise the vector nature of the control signal, and is again covered by the general formulation we propose.

I hope this answers your questions!
